src/scripts/main.js

```diff
@@ -1 +1,21 @@
 'use strict';
+
+window.addEventListener('hashchange', () => {
+  if (window.location.hash === '#menu') {
+    document.body.classList.add('page__body--with-menu');
+  } else {
+    document.body.classList.remove('page__body--with-menu');
+  }
+});
+
+const toTopBtn = document.getElementById('topButton');
```

Comment:
If you want to use some elements in JS, you better add `js-{some-name}` class and use it in your script.
Here is better do smth like this:
```
const toTopBtn = document.querySelector('.js-top-button');
```